internal/pkg/cli/svc_deploy.go

```diff
@@ -103,7 +115,20 @@ func newSvcDeployOpts(vars deployWkldVars) (*deploySvcOpts, error) {
 		},
 		cmd:          exec.NewCmd(),
 		sessProvider: sessions.NewProvider(),
-	}, nil
+	}
+	opts.uploadOpts = &uploadCustomResourcesOpts{
```

Comment:
do we need to update `cli/deploy.go` as well? it'd be rad if that file could just use these constructors again so that we don't have to remember to update the fields in multiple locations.